com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -46,103 +48,56 @@ internal enum NExec
         // RuntimeAccessModifiersILPP will make this `protected`
         internal BitWriter BeginSendServerRpc(ServerRpcSendParams sendParams, bool isReliable)
         {
-            // @mfatihmar (Unity) Begin: Temporary, placeholder implementation
-            var stream = new BitStream();
-            var writer = new BitWriter(stream);
+            var rpcQueueMananger = NetworkingManager.Singleton.RpcQueueManager;
```

Comment:
Why are we checking for null here? its a Singleton and it seems inconsistence in its checking. Also if you are going to check then why not use null coalescence?

Reply:
yeah +1 for this. if we're already here, we surely have a valid `NetworkingManager.Singleton` instance and it should already have a valid `RpcQueueManager` instance too. (very little thing to nit-pick but I think we should call it something other than `Manager`, maybe `Container`, `Component` etc. but I'm OK either ways)

Reply:
Yeah,  I have to agree that if there is no instance then something else is very wrong and should most likely naturally throw an exception.  I will add this to a new RPCQueue Revisions Jira Task.

Reply:
(I think renaming the RpcQueueManager to RpcQueueContainer is a good choice and will add that to the new task as well).